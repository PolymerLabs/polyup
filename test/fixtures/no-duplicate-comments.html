<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='duplicate-comments'>
  <script>
    Polymer({
      ready: function() {
        // Ready function.
        var ready = true;

        /* This is an example block comment which is intended
         * to refer to the block of code below. */
        var bar = 2;

        // This is a line comment which refers to the block below.
        var baz = 3;
      }
    })
  </script>
</polymer-element>
