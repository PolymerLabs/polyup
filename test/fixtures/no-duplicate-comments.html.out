<link rel="import" href="../polymer/polymer.html">

<dom-module id="duplicate-comments">
  <script>
    Polymer({
      is: 'duplicate-comments',
      ready: function () {
        // Ready function.
        var ready = true;
        /* This is an example block comment which is intended
         * to refer to the block of code below. */
        var bar = 2;
        // This is a line comment which refers to the block below.
        var baz = 3;
      }
    });
  </script>
</dom-module>
